src/components/BackScreen.tsx

```diff
@@ -22,7 +22,7 @@ const BackScreen = ({
       backgroundColor={backgroundColor || 'primaryBackground'}
       flex={1}
     >
-      <BackButton paddingHorizontal="lx" onPress={navigation.goBack} />
+      <BackButton marginHorizontal="n_s" onPress={navigation.goBack} />
```

Comment:
what is `n_s` and `lx` ?

Reply:
n stands for negative, so negative_small. lx is in between large and extra large.

```
  spacing: {
    n_xxl: -60,
    n_xl: -40,
    n_lx: -32,
    n_l: -24,
    n_lm: -20,
    n_m: -16,
    n_ms: -12,
    n_s: -8,
    n_xs: -4,
    n_xxs: -2,
    n_xxxs: -1,
    none: 0,
    xxxs: 1,
    xxs: 2,
    xs: 4,
    s: 8,
    ms: 12,
    m: 16,
    lm: 20,
    l: 24,
    lx: 32,
    xl: 40,
    xxl: 60,
  }
```